ports/net/macathena-attachandrun/Portfile

```diff
@@ -1,25 +1,10 @@
 # $Id$
 PortSystem		1.0
+PortGroup		obsolete 1.0
 
 name			macathena-attachandrun
-version			20060626
-distname		attachandrun-${version}
+replaced_by		macathena-athrun
+version			20060626.1
```

Comment:
You should add
```
revision 1
```
instead of changing the version (since .1 is not part of the version)